test/widgets/action_sheet_test.dart

```diff
@@ -38,11 +38,14 @@ late FakeApiConnection connection;
 Future<void> setupToMessageActionSheet(WidgetTester tester, {
   required Message message,
   required Narrow narrow,
+  Account? account,
 }) async {
   addTearDown(testBinding.reset);
 
-  await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot());
-  final store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
+  account ??= eg.selfAccount;
+  final initialSnapshot = eg.initialSnapshot(zulipFeatureLevel: account.zulipFeatureLevel);
```

Comment:
Does this need to start mentioning zulipFeatureLevel? If not, keep the story simpler by leaving it out.